yarn.lock

```diff
@@ -1,6086 +1,3878 @@
-# This file is generated by running "yarn install" inside your project.
```

Comment:
I don't think you need to check in this change, because you haven't made any yarn changes in the code
when I checked it in, it caused the pipeline to fail at the smoke test step